# Copyright 2023 TIER IV, Inc.
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.

/**:
  ros__parameters:
    publish_frame_count: 50
    heatmap_frame_id: "base_link"
    heatmap_length: 200.0
    heatmap_resolution: 0.8
    use_confidence: false
    class_names:
      [
        "UNKNOWN",
        "CAR",
        "TRUCK",
        "BUS",
        "TRAILER",
        "BICYCLE",
        "MOTORBIKE",
        "PEDESTRIAN",
      ]
    rename_to_car: false
